styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -84,7 +84,7 @@ public class FlyteAdminClientRunner implements FlyteRunner {
   @VisibleForTesting static final String STYX_WORKFLOW_INSTANCE_ANNOTATION = "styx-workflow-instance";
   @VisibleForTesting static final String STYX_EXECUTION_ID_ANNOTATION = "styx-execution-id";
   @VisibleForTesting static final Duration TERMINATION_GRACE_PERIOD = Duration.ofMinutes(10);
-  private static final Duration TERMINATION_LOOKUP_SINCE = Duration.ofDays(24);
```

Comment:
😮 Are are sure this was not a bug? `24 days` looks too much to be true, and the number `24` feels like hours anyway.

Reply:
yeah i think it was unintentional